admin/src/App.vue

```diff
@@ -53,6 +53,7 @@ export default Vue.extend({
     return {
       choices: this.$store.state.agents,
       selectedAgent: '',
+      activeName: 'try-it',
```

Comment:
activeName should be initiated by an empty string to avoid selecting try-it tab after needed one